React_Day4/avengers/src/Components/AppLayout.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import CardComponent from './CardComponent'
+import { data } from '../data';
+
+export default function AppLayout() {
+  return (
+    <div id='applayout' className='applayout-class'>
+        <CardComponent avenger={data[0]} />
+        <CardComponent avenger={data[1]} />
+        <CardComponent avenger={data[2]} />
+        <CardComponent avenger={data[3]} />
+        <CardComponent avenger={data[4]} />
+        <CardComponent avenger={data[5]} />
+        <CardComponent avenger={data[6]} />
+        <CardComponent avenger={data[7]} />
+        <CardComponent avenger={data[8]} />
+        <CardComponent avenger={data[9]} />
+    </div>
+  )
+}
+
+// export default function AppLayout() {
+//     return (
+//         data.map((i, index)=>(
+//             <CardComponent avenger={i} key={index} />
+//         ))
+//     )
+// }
+
+/*

```

Comment:
Use Better Comments extension.
Use multiline comments like this-
```
/*
* some comment1
* some comment2
* /
```

OR

```
/*
* * some comment1
* * some comment2
* /
```

Use function description as 
```
/*
* *  @param {string} str1 first string
* *  @param {string} str2 second string
* /  @returns {string} concatenated string

function concatenate (str1, str2){
    return str1 + str1
}
```